src/Orleans/Runtime/GrainMethodInvocationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.CodeGeneration;
+using Orleans.Concurrency;
+
+namespace Orleans.Runtime.GrainMethodInvocationExtensions
+{
+    public static class GrainMethodInvocationExtensions
```

Comment:
The reason why it's not there is https://github.com/dotnet/orleans/issues/2893#issuecomment-292002821. If we don't actually worry about that - will move.